docs/user/reporting/reporting-troubleshooting.asciidoc

```diff
@@ -164,3 +158,22 @@ https://www.elastic.co/elastic-maps-service[{ems} ({ems-init})] is a service tha
 tile layers and vector shapes of administrative boundaries.
 If a report contains a map with a missing basemap layer or administrative boundary, the {kib} server does not have access to {ems-init}.
 See <<maps-connect-to-ems>> for information on how to connect your {kib} server to {ems-init}.
+
+[float]
+[[reporting-manual-chromium-install]]
+=== Manually Installing the Chromium browser for Darwin
+Chromium is not embedded into {kib} for the Darwin (Mac OS) architecture. When
+running {kib} on Darwin, Reporting will download Chromium into proper area of
+the {kib} installation path the first time the server starts. If the server
+does not have access to the Internet, you will need to manually download the
+Chromium browser separately and install it into the {kib} installation path.
+Here are the steps to do that:
+
+1. Download the Chromium zip file.
+
+** For x64 systems, the link is: https://commondatastorage.googleapis.com/chromium-browser-snapshots/Mac/901912/chrome-mac.zip
```

Comment:
how about not showing the link and doing this instead:

1. Download the Chromuim zip file:
   - For [x64 systems](https://commondatastorage.googleapis.com/chromium-browser-snapshots/Mac/901912/chrome-mac.zip)
   - For [ARM systems](https://commondatastorage.googleapis.com/chromium-browser-snapshots/Mac_Arm/901913/chrome-mac.zip)